invenio_previewer/assets/semantic-ui/js/invenio_previewer/open_pdf.js

```diff
@@ -1,10 +1,94 @@
-// This file is part of InvenioRDM
-// Copyright (C) 2022 TU Wien.
-//
-// Invenio Theme TUW is free software; you can redistribute it and/or modify it
-// under the terms of the MIT License; see LICENSE file for more details.
-
-document.addEventListener("DOMContentLoaded", function () {
-  var fileUri = document.getElementById("pdf-file-uri").value;
-  window.PDFView.open(fileUri);
+/* Copyright 2014 Mozilla Foundation
+ * Copyright 2024 TU Wien
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
most of the file is copied over from the pdfjs repository, with the file originally being licensed under apache-2.0: https://www.tldrlegal.com/license/apache-license-2-0-apache-2-0
with it being a permissive FOSS license, i don't think there should be any licensing issues